### PR TITLE
Releasenotes 5.2.1

### PR DESCRIPTION
**Pull request by tbsbdr**
Added releasenotes
- Enhanced Functionality
- Notable Bugfixes
- Requirements
- Newly supported platforms
- Deprecated
- Deprecation Announcements
- Branding

### CODE REVIEW

File: modules/ROOT/pages/appendices/release_notes.adoc [hunk 1]
```diff
@@ -4,3 +4,114 @@
...
+
+The ownCloud Desktop Client is now available in version 5.2.1! If you have an enterprise subscription, this is the first 5.x release available for branding. We strongly recommend updating the Desktop Sync Client to the latest available version so that you can access and experience a range of new features and security fixes. Users with an Enterprise subscription are entitled to receive dedicated support.
+
+=== Enhanced Functionality
+
+* Store proxy password securely
+* Windows VFS: Prevent rename to ignored file name
+* Change how all files deleted is handled
+* Enable crash reporter in commandline client
+* Log http request when it is send
```
Comment (phil-davis) [inline modules/ROOT/pages/appendices/release_notes.adoc:18]:
```suggestion
* Log http request when it is sent
```

I would guess a bit about this, does it mean "Log the time that each http request is sent" or a general "Log http requests" feature (that log them somewhere in the client "when they are sent".

File: modules/ROOT/pages/appendices/release_notes.adoc [hunk 2]
```diff
@@ -4,3 +4,114 @@
...
+*	Authentication dialog no longer appears again and again
+*	Possible deadlock during log setup
+*	Do not create default sync root when loading accounts
+*	Update capabilites and other info after connect
+*	Account activity and crash after an account was removed
+*	Crash during application shutdown
+*	Hide hidden folders again in the selective sync view
+*	Fix check if a file is a placeholder
+*	Hydration state of file after a directory was replaced with a file
+*	Only syncronize after the server settings where refreshed
```
Comment (phil-davis) [inline modules/ROOT/pages/appendices/release_notes.adoc:51]:
```suggestion
*	Only syncronize after the server settings were refreshed
```

File: modules/ROOT/pages/appendices/release_notes.adoc [hunk 3]
```diff
@@ -4,3 +4,114 @@
...
+*	Consitently use the same icon for folders
+*	Branding of folder status overlay
+*	Creation of folder on the server
+*	Selective sync when not the remote root is synced
```
Comment (phil-davis) [inline modules/ROOT/pages/appendices/release_notes.adoc:55]:
```suggestion
*	Selective sync when other than the remote root is synced
```

Maybe this is a way to say it?

File: modules/ROOT/pages/appendices/release_notes.adoc [hunk 4]
```diff
@@ -4,3 +4,114 @@
...
+*	Fedora 37
+*	Fedora 38
+*	Fedora 39
+*	openSUSE Leap 15.4
+*	openSUSE Leap 15.5
+*	Ubuntu 22.04 LTS
+*	Ubuntu 23.04
+*	Ubuntu 23.10
+
+AppImage builds for other Linux distributions Here you can find the full requirements page.
```
Comment (phil-davis) [inline modules/ROOT/pages/appendices/release_notes.adoc:81]: Something is not right here?

File: modules/ROOT/pages/appendices/release_notes.adoc [hunk 5]
```diff
@@ -4,3 +4,114 @@
...
+
+Looking further, the following versions may no longer be supported in future releases:
+
+*	macOS 10.15 Catalina
+*	Fedora 37
+
+=== Branding
+
+If you're entitled to create branded versions of the ownCloud Desktop Client, please visit customer.owncloud.com to start the branding process for 5.2.1
+Updater Server 1.1.0
```
Comment (phil-davis) [inline modules/ROOT/pages/appendices/release_notes.adoc:114]: Should this be a heading or?

File: modules/ROOT/pages/appendices/release_notes.adoc (hunk 4 above)
Comment (tbsbdr) [reply modules/ROOT/pages/appendices/release_notes.adoc:81]: Hey Phil, what do you mean?

Comment (TheOneRing) [reply modules/ROOT/pages/appendices/release_notes.adoc:81]: The uppercase Here probably.

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**